src/main/java/hudson/plugins/ws_cleanup/Cleanup.java

```diff
@@ -129,7 +131,7 @@ public Object invoke(File f, VirtualChannel channel) throws IOException, Interru
         } else {
             tempCommand = deleteCommand + " " + fullPath;
         }
-        List<String> cmdList = new ArrayList<String>();
+        List<String> cmdList = new ArrayList<>();
```

Comment:
:-1:
I'm supporting the original form.

Reply:
Having to provide the same type twice is just code duplication.
But I can remove the commit https://github.com/jenkinsci/ws-cleanup-plugin/pull/42/commits/8d39e5f7a319016776a9e69682979000aaff35c4 which introduced the diamond operator.
That's exactly why the pull request is split into many commits - to get a feedback on the different aspects.